package.json

```diff
@@ -70,7 +70,8 @@
     "karma-sourcemap-loader": "^0.3.4",
     "karma-spec-reporter": "0.0.16",
     "karma-webpack": "~1.5.0",
-    "mocha": "^2.1.0"
+    "mocha": "^2.1.0",
+    "webpack-bundle-tracker": "0.0.8"
```

Comment:
JSON files won't allow you to have the trailing comma on the last line because it's not considered proper syntax and results in parsing errors.